heartbeat/LVM-activate

```diff
@@ -701,7 +707,7 @@ tagging_deactivate() {
 # method:
 #
 # lv_count=$(vgs --foreign -o lv_count --noheadings ${VG} 2>/dev/null | tr -d '[:blank:]')
-# dm_count=$(dmsetup --noheadings info -c -S "vgname=${VG}" 2>/dev/null | grep -c "${VG}-")
```

Comment:
This and the other `vgname` to `vg_name` / `lvname` to `lv_name` should probably be a separate commit, and make sure to write a commit message to explain this change: Is it a simple typo, or is it a change between versions? If it is the latter, there should be a version check to use the old name with older versions.